Content.Shared/GameObjects/ContentNetIDs.cs

```diff
@@ -68,7 +68,7 @@ public static class ContentNetIDs
         public const uint BOLTACTION_BARREL = 1062;
         public const uint PUMP_BARREL = 1063;
         public const uint REVOLVER_BARREL = 1064;
-	public const uint CUFFED = 1065;
+	    public const uint CUFFED = 1065;
```

Comment:
You uh really like that indentation huh?

Reply:
? 

in vs it's not lining up with the rest of the IDs